lib/definition_parser.rb

```diff
@@ -0,0 +1,48 @@
+class DefinitionParser
```

Comment:
can we put example & explanation somewhere (probably in README) that shows definition

Reply:
Sure Mas, We will add this in README.

Reply:
Done mas @giosakti, please check. It's on `doc/` folder, named `deployments.md`.